flocker/acceptance/testtools.py

```diff
@@ -938,7 +948,8 @@ def check_http_server(host, port):
     """
     req = get(
```

Comment:
Started down that path, found some (weak) reasons not to do that (not many callers, not suitable for all callers), decided not worth bothering at this point.  Might make sense later ... although maybe later we'll have escaped the gravity of the container agent.